src/plugins/telemetry_collection_manager/server/plugin.ts

```diff
@@ -50,12 +50,12 @@ export class TelemetryCollectionManagerPlugin
   private readonly collections: Array<Collection<any>> = [];
   private usageGetterMethodPriority = -1;
   private usageCollection?: UsageCollectionSetup;
-  private readonly isDev: boolean;
+  private readonly isProd: boolean;
   private readonly version: string;
 
   constructor(initializerContext: PluginInitializerContext) {
     this.logger = initializerContext.logger.get();
-    this.isDev = initializerContext.env.mode.dev;
+    this.isProd = initializerContext.env.packageInfo.dist; // Is it a distributable release? => isProd
```

Comment:
"prod" and "dev" mean something different thank "dist" and "source", this probably shouldn't be called `isProd`